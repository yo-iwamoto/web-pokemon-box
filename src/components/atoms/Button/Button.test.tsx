import { Button } from '.';
import { render } from '@testing-library/react';

describe('Button', () => {
  it('正しく表示されること', () => {
    const { getByRole, asFragment } = render(<Button type='button'>click me</Button>);

    expect(getByRole('button', { name: 'click me' })).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
