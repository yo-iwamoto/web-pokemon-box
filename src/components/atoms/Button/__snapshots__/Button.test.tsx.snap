// Vitest Snapshot v1

exports[`Button > 正しく表示されること 1`] = `
<DocumentFragment>
  <button
    class="rounded bg-blue-500 py-2 px-4 font-bold text-white transition-colors hover:bg-blue-400"
    type="button"
  >
    click me
  </button>
</DocumentFragment>
`;
